docs/EKS.md

```diff
@@ -414,7 +414,7 @@ You can configure your scaling group to use [AWS Warm Pools for Auto Scaling](ht
 Warm Pools is not officially supported with EKS, hence the following requirements exist in order to use it with EKS:
 - Your AMI must have `awscli` & `jq`, you can also install it in pre-bootstrap userdata.
```

Comment:
We may want to modify this doc as well to say `awscli` & `jq` are needed in the case of AL2, pretty sure we also need awscli for windows nodes

Reply:
Clarified this - for Windows, the Get-ASAutoScalingInstance powershell cmdlet is the only dependency.